app/views/spree/shared/_google_analytics.html.erb

```diff
@@ -1,36 +1,42 @@
-<% if tracker = Spree::Tracker.current(current_store) %>
-	<script id="solidus_trackers">
-		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-		})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+<% trackers = Spree::Tracker.current(Spree::Store.default, 'analytics') %>
 
-		ga('create', '<%= tracker.analytics_id %>', 'auto');
-		ga('require', 'displayfeatures');
-		ga('send', 'pageview');
+<% if trackers.any? %>
+  <% trackers.each do |tracker| %>
```

Comment:
Do you mind taking a look at the indentation in this file? It looks really odd.

Reply:
done https://github.com/solidusio-contrib/solidus_trackers/pull/10/commits/4b43e34398e2537b26c73861df24c1c1ac3f5f0c#diff-8487831e71c4c9024a2919aa6473233aR3